app-config.yaml

```diff
@@ -434,8 +432,10 @@ costInsights:
       kind: 'PINTS_OF_ICE_CREAM'
       unit: 'ice cream pint'
       rate: 5.5
-pagerduty:
+
+pagerDuty:
   eventsBaseUrl: 'https://events.pagerduty.com/v2'
```

Comment:
Updated with capital "D" to match config: https://github.com/backstage/backstage/blob/1a14adeb574f7846b6d5f5ef555a57dea403e03a/plugins/pagerduty/config.d.ts#L21